components/table/src/index.js

```diff
@@ -107,13 +151,13 @@ const Table = ({ name, names, rowIncludesHeading, titles, rows, flexibleColumns,
           {row.map(
             (item, itemIndex) =>
               rowIncludesHeading && itemIndex === 0 ? (
-                <TableHeading rowHeading columnCount={row.length} key={item.key || itemIndex} verticalAlign={verticalAlign} name={names.headings}>
+                <TableHeading rowHeading columnCount={row.length} key={item.key || itemIndex} name={getName(names, calculateIndex(titles, nameByRow, index), itemIndex, nameByRow)}>
```

Comment:
this line is far too wide and should be triggering a linting error as it's over 120 chars long :grin:

Reply:
Let's make a call on https://github.com/stevesims/govuk-frederic/issues/5 when we can. Also on whether to introduce Prettier. Maybe Ali is looking at this side of things. I think Prettier may be worth it but we'd need to be prepared to see a few commits which just change formatting.